src/pkg/utils/arch.go

```diff
@@ -25,6 +25,11 @@ func getFullyQualifiedImageArch(image, release string) string {
 	return imageFull
 }
 
+func getP11KitClientPathsArch() []string {
+	paths := []string{"/usr/lib/pkcs11/p11-kit-client.so"}
```

Comment:
@Foxboron I hope I got the path to `p11-kit-client.so` right for Arch Linux.  I suppose we build the `arch-toolbox:latest` image only for `x86_64`, correct?

Reply:
```
λ ~ » sudo pacman -F p11-kit-client.so
core/libp11-kit 0.25.5-1 [installed]
    usr/lib/pkcs11/p11-kit-client.so
```

The path is correct :)

> I suppose we build the arch-toolbox:latest image only for x86_64, correct?

Yes

Reply:
Thanks for confirming that!